roles/kustomize_deploy/README.md

```diff
@@ -0,0 +1,49 @@
+# kustomize_deploy
+
+Ansible role designed to deploy VA scenarios using the kustomize tool.
+
+## Parameters
+
+- `cifmw_kustomize_deploy_destfiles_basedir`: Base directory for the ci-framework artifacts.
+  Defaults to `~/ci-framework-data/`
+- `cifmw_kustomize_deploy_architecture_repo_url`: URL of the "architecture" repository, where the VA scenarios are defined.
+  Defaults to `https://github.com/openstack-k8s-operators/architecture`
+- `cifmw_kustomize_deploy_architecture_repo_dest_dir`: Directory where the architecture repo is cloned on the controller node.
+  Defaults to `~/src/github.com/openstack-k8s-operators/architecture`
+- `cifmw_kustomize_deploy_architecture_repo_version`: Default branch of the architecture repo to clone.Defaults to `HEAD`
+- `cifmw_kustomize_deploy_kustomizations_dest_dir`: Path for the generated CR files.
+  Defaults to `stomize_deploy_destfiles_basedir + /artifacts/kustomize_deploy`
```

Comment:
typo, should be `cifmw_kustomize_deploy_destfiles_basedir + /artifacts/kustomize_deploy`

Reply:
ah! Thanks a lot @cescgina !